src/DependencyManagement/DependencyManager.cs

```diff
@@ -418,201 +270,51 @@ internal string GetModuleLatestSupportedVersion(string moduleName, string majorV
             return latestVersion;
         }
 
-        /// <summary>
-        /// Parses the given string version and extracts the major version.
-        /// Please note that the only version we currently support is of the form '1.*'.
-        /// </summary>
-        private string GetMajorVersion(string version)
+        private void WaitOnDependencyInstallationTask()
```

Comment:
What would you want do see documented, except for something like "Wait until the dependency installation task is not running anymore"? Can this be expressed in the method name? If it can, this is what I would rather do with private methods.

Reply:
The method name is consistent with what's happening in the code. Conversation resolved.